pom.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
Question for you @davewichers - This is just an XML namespace and often IRL, these URLs do not even exist and would return a 404. So if we were to change them to https would the schema validation still work or is there a chance it could break something. TBH, I've never seen https:// schema used in XML namespace and I'd be concerned if someone wrote a parser to do schema validations of Maven pom files (like might be in some IDEs), that this change could break something.

Reply:
@davewichers - I'm going to reject this particular change as it causes an error / warning in Eclipse and perhaps other zIDEs as well. See attached pic for example
![ESAPI-pom-Eclipse](https://user-images.githubusercontent.com/1959835/185767125-f336faa7-279a-4d5b-b001-3442bbcbf148.jpg)
.